cmd/downlaod_mgr.go

```diff
@@ -223,3 +224,14 @@ func copyFile(src, dst string) error {
 
 	return nil
 }
+
+func ParseTime(dateString string) time.Time {
```

Comment:
This is not really a general function,,, 
I think we need a strucut for handling downloader-manager.